csvcubed/csvcubed/readers/cubeconfig/v1_0/columnschema.py

```diff
@@ -98,35 +100,40 @@ def _get_code_list(self, new_dimension: NewQbDimension) -> Optional[QbCodeList]:
             elif (
                 new_dimension.parent_dimension_uri
                 == "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod"
-                and self.definition_uri is not None
-                and self.definition_uri.lower().startswith(
+                and self.cell_uri_template is not None
+                and self.cell_uri_template.lower().startswith(
                     "http://reference.data.gov.uk/id/"
```

Comment:
I suggest we define urls like these in a constants or a config file. In this way, if we want to change these, we can easily do so.